sulu/SuluAutomationBundle
# Fixed negative notification badge

This PR fixes negative notification badge when deleting task.

How to reproduce:

1. Open the same page in two tabs
2. Create a new task in one of this tabs
3. Without reloading go in the second tab on the automation tab => you should see a task but no notification-badge
4. Delete task => you should see `-1` badge
